src/utils/firebaseConfig.ts

```diff
@@ -0,0 +1,24 @@
+import { initializeApp } from "firebase/app";
+import {
+  getFirestore,
+  collection,
+  onSnapshot,
+  addDoc,
+  deleteDoc,
+  doc,
+} from "firebase/firestore";
+
+const firebaseConfig = {
```

Comment:
Do not publicized sensitive data in the source file. Use a separate .env file to store them and access from there